docs/user-guide/src/irso/introduction.md

```diff
@@ -0,0 +1,72 @@
+# Ironic Standalone Operator
+
+Ironic Standalone Operator (IrSO) is a Kubernetes controller that installs and
+manages Ironic in a configuration suitable for Metal3. IrSO has the following
+features:
+
+- Flexible networking configuration, support for Keepalived.
+- Using SQLite or MariaDB as the database backend.
+- Optional support for a DHCP service (dnsmasq).
+- Optional support for automatically downloading an
+  [IPA](../ironic/ironic-python-agent.md) image.
+
+IrSO uses [ironic-image](../ironic/ironic-container-images.md) under the hood.
+
+## Installing Ironic Standalone Operator
+
+On every source code change, a new IrSO image is built and published at
+`quay.io/metal3-io/ironic-standalone-operator`. To install it in your cluster,
+you can use the Kustomize templates provided in the source repository:
+
+```console
+git clone https://github.com/metal3-io/ironic-standalone-operator
+cd ironic-standalone-operator
+git checkout -b <DESIRED BRANCH OR main>
+
+make install deploy
+kubectl wait --for=condition=Available --timeout=60s \
+  -n ironic-standalone-operator-system deployment/ironic-standalone-operator-controller-manager
+```
+
+## API resources
+
+IrSO uses two Custom Resources to manage an Ironic installation:
+
+[Ironic](https://github.com/metal3-io/ironic-standalone-operator/blob/main/config/crd/bases/ironic.metal3.io_ironics.yaml)
+manages Ironic itself and all of its auxiliary services.
+
+[IronicDatabase](https://github.com/metal3-io/ironic-standalone-operator/blob/main/config/crd/bases/ironic.metal3.io_ironicdatabases.yaml)
+manages a MariaDB instance for Ironic (if required).
+
+See [installing Ironic with IrSO](./install-basics.md) for information on how
+to use these resources.
+
+## How is Ironic installed?
+
+By default, IrSO installs Ironic as a single pod on a **control plane** node.
+This is because Ironic currently requires *host networking*, and thus it's not
+advisable to let it co-exist with tenant workload.
+
+### Installed services
```

Comment:
Coming from a kubernetes background, this section can be a bit confusing. The terminology is easy to misunderstand since some of the words have specific meanings in kubernetes context (e.g. service and deployment). That said, I'm not quite sure how I would change it :grimacing: 

Is it so that all of these below are containers in the ironic pod? I would then maybe prefer to talk about them as "ironic sidecars". What do you think?

Reply:
Perhaps 'An Ironic instance (?) always contains these three components:'